src/overlays/actors/ovl_En_Okarina_Tag/z_en_okarina_tag.c

```diff
@@ -30,14 +29,123 @@ const ActorInit En_Okarina_Tag_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+void EnOkarinaTag_Destroy(Actor* thisx, PlayState* play) {
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Okarina_Tag/EnOkarinaTag_Destroy.s")
+void EnOkarinaTag_Init(Actor* thisx, PlayState* play) {
+    EnOkarinaTag* this = THIS;
+    f32 zRot;
+    s32 i;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Okarina_Tag/EnOkarinaTag_Init.s")
+    zRot = 0.0f;
+    i = 0;
+    this->actor.flags &= ~1;
```

Comment:
```suggestion
    this->actor.flags &= ~ACTOR_FLAG_1;
```
We have flag defines for actor flags, I think Maide had a script that can automatically fill them in for you, but in this case it should be 
`this->actor.falgs &= ~ACTOR_FLAG_1;` since they aren't named yet (although we know 1 is "player can Z target this actor")